js/client-chat.js

```diff
@@ -28,7 +28,7 @@
 			var name = app.user.get('name');
 			var userid = app.user.get('userid');
 			if (this.expired) {
-				this.$chatAdd.text(this.expired === true ? 'This room is expired' : this.expired);
+				this.$chatAdd.html(this.expired === true ? 'This room is expired' : this.expired);
```

Comment:
Eek, where is the validation? This makes me uncomfortable...

Reply:
Where's the validation for any of them?

Reply:
`.text` doesn't need validation, `.html` is an XSS vector. We generally run `sanitizeHTML` or `escapeHTML` on any untrusted data we receive, before embedding it in HTML.